migration/migrator/migrations/course/20240208120646_custom_answers_polls.py

```diff
@@ -0,0 +1,40 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute(
+        """
+        ALTER TABLE polls
+        ADD COLUMN IF NOT EXISTS allows_custom boolean NOT NULL DEFAULT false;
+
+        ALTER TABLE poll_options
+        ADD COLUMN IF NOT EXISTS author_id VARCHAR(255) NOT NULL DEFAULT 'instructor';
```

Comment:
As I imagine it, every instructor will have an author_id of 'instructor', while students will have an author_id of their RCSID.

Reply:
Discussed in meeting. Switched default value to an empty string in [1f1eb00](https://github.com/Submitty/Submitty/pull/10197/commits/1f1eb00c946562470f9fa960ab6f1e856ec721a6). Null values don't work very well in this scenario since PHP likes to change them to empty strings.